core/solidus_core.gemspec

```diff
@@ -29,8 +29,8 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activemerchant', '~> 1.66'
   s.add_dependency 'acts_as_list', '~> 0.3'
-  s.add_dependency 'cancancan', '~> 2.2'
   s.add_dependency 'awesome_nested_set', '~> 3.2'
+  s.add_dependency 'cancancan', ['>= 2.2', '< 4.0']
```

Comment:
Just saw that we should update the commit message

Reply:
@tvdeyen you're right, it wasn't very clear. What about now?

Reply:
:+1: thanks